docs/operations/default_seccomp_profile_seed.md

```diff
@@ -32,25 +32,3 @@ featureGates:
 Please refer to the examples in this [yaml file](../../example/20-componentconfig-gardenlet.yaml) for more information.
 
 Once the feature gate is enabled, the webhook will be registered and configured for the seed cluster. Newly created pods will be mutated to have their seccomp profile set to `RuntimeDefault`.
-
-> **Note:** Please note that this feature is still in Alpha, so you might see instabilities every now and then. 
```

Comment:
Why is this sentence removed? To my knowledge the `DefaultSeccompProfile` feature gate of gardenlet is still alpha.

Reply:
It was removed after asking @dimityrmirchev about it's relevance

Reply:
Yes, the feature is in alpha but pods will be mutated reliably with `RuntimeDefault` seccomp profile. I am not sure that there would be any instabilities w.r.t. the feature being `Alpha`.